browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchClient.java

```diff
@@ -91,57 +91,35 @@ private boolean isValidRequestCode(int requestCode) {
     }
 
     /**
-     * Deliver a pending browser switch result to an Android activity that is also a BrowserSwitchListener.
```

Comment:
```
* Deliver a pending browser switch result to an Android activity
```

Don't we want to keep/modify this first sentence since it describes what the method actually does?

Reply:
Yeah we can re-structure it a bit. The main reason for deletion here was because `BrowserSwitchListener` is no longer needed since the `deliverResult` method is now a synchronous call.